keps/sig-cluster-lifecycle/clusterapi/README.md

```diff
@@ -0,0 +1,4 @@
+# Cluster API CAEPs
+
+Cluster API follows its [own process](https://cluster-api.sigs.k8s.io/contributing#proposal-process-caep) for enhancement proposals.
```

Comment:
Could we call out here that the CAEP process is fundamentally a derivative of the KEP?

Reply:
Sure that would be helpful. Should I add it as a note?

Reply:
I feel like other than the comment of CAEP, it looks nice.